kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/UpsertDeleteValidator.java

```diff
@@ -61,6 +65,14 @@ protected Optional<String> doValidate(BigQuerySinkConfig config) {
       ));
     }
 
+    if (mergeInterval != -1 && mergeInterval < 10_000L) {
```

Comment:
can we write UT to cover test line coverage for this if block?

Reply:
As discussed, this is a warning logger and the logger is declared as private static final which is not possible to mock in mockito. So skipping this.